### PR TITLE
feat: request-float-license

### PR DESCRIPTION
**Pull request by muneebkq**

### CODE REVIEW

File: cryptlex/lexfloatclient/lexfloatstatus_codes.py
Comment (ahmad-kemsan) [inline cryptlex/lexfloatclient/lexfloatstatus_codes.py:1]: @muneebkq Why these changes?

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please check the comments

File: cryptlex/lexfloatclient/lexfloatstatus_codes.py
Comment (muneebkq) [reply cryptlex/lexfloatclient/lexfloatstatus_codes.py:1]: This format is followed in every other function.

File: cryptlex/lexfloatclient/lexfloatstatus_codes.py [hunk 1]
```diff
@@ -1,59 +1,73 @@
...
+    
     LF_E_LICENSE_EXISTS = 46
-
+    
     LF_E_LICENSE_NOT_FOUND = 47
-
+    
     LF_E_LICENSE_EXPIRED_INET = 48
-
+
```
Comment (ahmad-kemsan) [inline cryptlex/lexfloatclient/lexfloatstatus_codes.py:24]: @muneebkq There are still many formatting issues that need to be resolved.

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please check the comment

**Review by ahmad-kemsan — APPROVED** [review]

**Pull request merged**